Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/MLproject

```diff
@@ -2,7 +2,8 @@ name: bert-classification
 
 docker_env:
   image: englabs/trainer
-  environment: [["MLFLOW_TRACKING_URI", "http://35.185.111.8:5000"]]
+  volumes: ["$(pwd):/tmp/englab/"]
+  environment: ["MLFLOW_TRACKING_URI"]
```

Comment:
I don't know if we need this any more, I think `mlflow run` is smart enough to use the host version itself if defined or use the one in the docker image if not

Reply:
Yes it is. However you gotta tell him to copy the variable if it exists in the host. That's why I let the _env_ there without its value.